app/static/js/components/Experiments/IngredientFrequency/IngredientFrequencyBarChart.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as d3 from "d3";
+import Axis from 'components/d3/Axis.jsx';
+import Bar from 'components/d3/Bar.jsx';
+import { withStyles } from '@material-ui/core/styles';
+
+
+class FrequencyBarChart extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { data: [], hasError: false }
+  }
+
+  componentWillMount() {
+    fetch('/api/ingredient_frequency')
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          data: data
+        })
+      })
+  }
```

Comment:
Note to self to add caching option down the row or a default if something with the request fails.